deepmd/pt/utils/tabulate.py

```
@@ -0,0 +1,920 @@
+    def _build_lower(
+        self, net, xx, idx, upper, lower, stride0, stride1, extrapolate, nspline
+    ):
+        vv, dd, d2 = self._make_data(xx, idx)
+        self.data[net] = np.zeros(
+            [nspline, 6 * self.last_layer_size], dtype=self.data_type
+        )
+
+        # tt.shape: [nspline, self.last_layer_size]
+        if isinstance(self.descrpt, deepmd.pt.model.descriptor.DescrptSeA) or isinstance(
+            self.descrpt, deepmd.pt.model.descriptor.DescrptDPA1
+        ):
```

Ruff

Multiple `isinstance` calls for expression, merge into a single call

Merge `isinstance` calls

(SIM101)